src/main/java/racingcar/Racing.java

```diff
@@ -0,0 +1,40 @@
+package racingcar;
+
+import racingcar.car.Car;
+import racingcar.car.NextStepCar;
+import racingcar.view.InputView;
+import racingcar.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Racing {
+    public static void main(String[] args) {
+        List<Car> cars = new ArrayList<>();
+
+        Scanner scanner = new Scanner(System.in);
+        InputView inputView = new InputView();
+        ResultView resultView = new ResultView();
+        inputView.print("자동차 대수는 몇 대 인가요?");
+        int carCount = scanner.nextInt();
+        inputView.print("시도할 회수는 몇 회 인가요?");
+        int forwartCount = scanner.nextInt();
+        Random random = new Random();
+
+        for (int i=0; i<carCount; i++) {
```

Comment:
단항 연산자(++ 혹은 --)의 경우와 .을 제외한 모든 이항(binary) 연산자는 연산수들과는 공백으로 분리하는 것이
컨벤션이오니 참고해주시면 감사하겠습니다 :)

```suggestion
        for (int i = 0; i < carCount; i++) {
```